README.md

```diff
@@ -1 +1,22 @@
-# WeatherApp
\ No newline at end of file
+# WeatherApp
```

Comment:
put the dates in the descendent mode pls

Reply:
Solved!